react/SelectionBar/index.jsx

```diff
@@ -82,7 +84,9 @@ const SelectionBar = ({ actions, selected, hideSelectionBar }) => {
             className={styles['SelectionBar-action']}
             label={t('SelectionBar.' + actionName)}
             key={index}
-            disabled={selectedCount < 1}
+            disabled={
+              selectedCount < 1 && !actions[actionName].enabledEvenEmpty
+            }
```

Comment:
Instead of adding a new `enabledEvenEmpty` action, why not writing something like: 

```jsx
actions = {
  trash: {
    action: selections => dispatch(trashFiles(selections))),
    icon: 'trash',
   disabled: false //or selections => selections.length < 1 
  },
  rename: {
    action: selections => dispatch(startRenamingAsync(selected[0])),
    displayCondition: selections => selections.length === 1
  }
}
```

By default, if there is no `disabled` method on the action, we use `selectedCount < 1`. 

WDYT?

Reply:
ça prête à confusion non ? Car on ne sait pas quand le `disabled` est interprété. De plus ça sous-entend que la valeur par défaut est `true` (et non `selectedCount < 1`) ce qui fait comprendre à tort que l'action n'est pas active du tout, à savoir qu'elle ne fonctionne pas.

Reply:
Dans ce cas on change disabled en enabled mais on doit avoir le full control dessus, comme ce qu'on a pour le displayCondition ou pour l'action. 

Toutes les actions devraient être définies en fonction de la sélection, pas en fonction de nouvelle "props" genre `enabledEvenEmpty`. L'API et l'usage en seront que plus simple.

Reply:
ok va pour `disabled` dans ce cas 👍

Reply:
De ce que je comprends maintenant, si tu peux mettre un exemple d'usage ce serait top, c'est que `disabled` est un boolean. 

J'imaginais plutôt une méthode, comme displayCondition qui reçoit en arguments les `selected` et qui en fonction de ça fait des choses. Comme ça les actions sont complètement autonomes et ce n'est pas à l'app qui appelle la selectionBar de récupérer la selection pour faire changer le boolean `disabled`

Reply:
C'est corrigé.